### DISCUSSION TITLE
PiVPN and FritzBox Wireguard - doubled up

### DISCUSSION THREAD

**Discussion by erpel13** [pivpn/pivpn#1880] [Q&A]
Hello,
I have a FritzBox 7590 with Wireguard installed.
This means I can access my home network via VPN when I'm on the go.
I've now also installed a Raspi with PiHole and am wondering whether it makes sense to install PiVPN on it?
Are there any exclusion criteria? Is that duplicated? Will everything still work properly?
Best regards

**Comment by orazioedoardo** [comment]
> Are there any exclusion criteria? 

What do you mean?

> Is that duplicated?

Mostly yes.

> Will everything still work properly?

Should work.

> I've now also installed a Raspi with PiHole and am wondering whether it makes sense to install PiVPN on it?

You can but you can also point WireGuard FritzBox client configs to the Raspi Pi-Hole DNS.
